alibi_detect/saving/_tensorflow/loading.py

```diff
@@ -36,36 +37,31 @@
 
 
 def load_model(filepath: Union[str, os.PathLike],
-               filename: str = 'model',
-               custom_objects: dict = None,
                layer: Optional[int] = None,
+               **kwargs
                ) -> tf.keras.Model:
     """
     Load TensorFlow model.
 
     Parameters
     ----------
     filepath
-        Saved model directory.
-    filename
-        Name of saved model within the filepath directory.
-    custom_objects
-        Optional custom objects when loading the TensorFlow model.
+        Saved model filepath. This should be a directory if the model is in the `SavedModel` format. Otherwise, it
+        should be a path to a `.h5` file.
     layer
         Optional index of a hidden layer to extract. If not `None`, a
         :py:class:`~alibi_detect.cd.tensorflow.HiddenOutput` model is returned.
-
+    kwargs
+        Additional keyword arguments to be passed to :func:`tf.keras.models.load_model`.
     Returns
     -------
     Loaded model.
     """
-    # TODO - update this to accept tf format - later PR.
-    model_dir = Path(filepath)
-    model_name = filename + '.h5'
-    # Check if model exists
-    if model_name not in [f.name for f in model_dir.glob('[!.]*.h5')]:
-        raise FileNotFoundError(f'{model_name} not found in {model_dir.resolve()}.')
-    model = tf.keras.models.load_model(model_dir.joinpath(model_name), custom_objects=custom_objects)
+    # Load model
+    model = tf.keras.models.load_model(filepath, **kwargs)
```

Comment:
Seems like we're throwing away the validation code for the existence of the model? Or is it done from higher up in another caller?

Reply:
OK that makes sense.